public/mobile-app/src/routes/+page.svelte

```diff
@@ -118,89 +152,92 @@ const registerWithAmi = async () => {
         p256dh: pushSubP256DH,
       },
     },
-    email: registerEmailValue,
+    email: registerEmailInputValue,
   }
-  console.log('isRegistered with AMI:', payload)
+  console.log('payload:', payload)
   isRegisteredWithAmi = true
-  registrationStatus = 'Registering...'
+  registrationStatus = "En cours d'enregistrement..."
 
   const response = await fetch(`${PUBLIC_API_URL}/notification/register`, {
     method: 'POST',
     body: JSON.stringify(payload),
   })
-  console.log('response from AMI:', response)
+  console.log('response:', response)
   isRegisteredWithAmi = false
   if (response.status < 400) {
-    registrationStatus = 'Done!'
-    userEmail = registerEmailValue
+    registrationStatus = 'Enregistrement réussi !'
+    userEmailState = registerEmailInputValue
+    userNotifications = []
+    window.localStorage.setItem('emailLocalStorage', userEmailState)
+    window.localStorage.setItem('pushSubscriptionLocalStorage', pushSubscription)
   } else {
     registrationStatus = `error ${response.status}: ${response.statusText}, ${response.body}`
   }
 }
 </script>
 
 <div>
-	{#if userEmail}
-	  <h1>Bienvenue {userEmail} sur l'application AMI</h1>
+  {#if userEmailState}
+    <h1>Bienvenue {userEmailState} sur l'application AMI</h1>
   {:else}
     <h1>Bienvenue sur l'application AMI</h1>
   {/if}
 
-	<button
-		type="button"
-		onclick={askForNotificationPermission}
-		disabled={isAuthenticatedForNotifications}
-	>
-		S'authentifier pour recevoir des notifications
-	</button>
-	<span id="subscription-status">{subscriptionStatus}</span>
-
-	<div>
-		<p>
-			<label
-				>Email
-				<input
-					bind:value={registerEmailValue}
-					type="text"
-					id="register-email"
-					name="register-email"
-				/>
-			</label>
-		</p>
-		<p>
-			<button
-				type="button"
-				id="register-with-ami"
-				onclick={registerWithAmi}
-				disabled={isRegisteredWithAmi}
-			>
-				S'enregistrer auprès d'AMI
-			</button>
-			<span id="registration-status">{registrationStatus}</span>
-		</p>
-	</div>
+  <button
+      type="button"
+      onclick={askForNotificationPermission}
+      disabled={isAuthenticatedForNotifications}
```

Comment:
This is nice. Would it make sense to go a step further, and only display the UI elements that are useful in any given state?
Eg: notifications are not authorized yet? Only display this button, nothing else.
Notifications are authorized? Don't display this button at all.

Email already registered?
- display a "register with another email" button
- hide the email form and the register button
Otherwise display the email form and register button, and hide the notifications list... and so on.

We could split this in other issues and PRs if it makes more sense.